schema/spacewalk/susemanager-schema.changes

```diff
@@ -1,8 +1,8 @@
+- Changed to versioned Python3 to SPEC file.
+- Python3 port for blend tool
 - internal code cleanup (dropping unused table rhnErrataTmp)
 - Fix Debian package version comparison
-- Use versioned Python2 for RHEL8 and Fedora.
 - Move dist upgrade SQL file to the correct directory so it gets picked up in schema upgrades (bsc#1179759)
-- Changed to versioned Python2 to SPEC file.
```

Comment:
Not sure if we should remove this. @juliogonzalez what is your take on this?

Reply:
Tested build successfully on CentOS 7, 8, Fedora 32 (33 and Rawhide fail due to Find.pm missing), Leap 15.2. - Reinstall also seems to works on CentOS 8.
So all green from my side.
If there has been no release after my previous commit, I would also prefer to remove the lines from the change log to avoid confusion. My previous commits would not show up at all in this version.